src/Token.php

```diff
@@ -26,6 +26,11 @@
     public function headers(): DataSet;
 
     /**
+     * Returns the token claims
+     */
+    public function claims(): DataSet;
```

Comment:
@lcobucci I think this was initially excluded due to encrypted tokens?

Reply:
@lcobucci well, we could add two interfaces `PlainToken` and `EncryptedToken` both extending `Token` and then add a public method `claims()` to `PlainToken` only. This solution is much cleaner in my opinion.

Reply:
@viteksafronov sorry for not getting back to you earlier. That's not ideal, I explained the reason also on #203